plugins/api-docs/src/components/ApiExplorerTable/ApiExplorerTable.tsx

```diff
@@ -123,8 +157,32 @@ export const ApiExplorerTable = ({
     );
   }
 
+  const rows = entities.map(e => {
+    const partOfSystemRelations = getEntityRelations(e, RELATION_PART_OF, {
+      kind: 'system',
+    });
+    const ownedByRelations = getEntityRelations(e, RELATION_OWNED_BY);
+
+    return {
+      ...(e as ApiEntityV1alpha1),
+      row: {
+        ownedByRelationsTitle: ownedByRelations
+          .map(r => formatEntityRefTitle(r, { defaultKind: 'group' }))
+          .join(', '),
```

Comment:
Do we still need these titles at all? I thought `EntityRefLinks` was handling it. Same below and in the other files. Or is it for filtering/sorting? In that case they should all use .map instead of taking just the first one

Reply:
Nope I just completely missed that...

Reply:
Haven't read it correctly. You are right, this is just for filtering/sorting. But yes, we should map all of them